lib/src/snippet/kotlin/com/lemonappdev/konsist/CleanArchitectureSnippets.kt

```diff
@@ -39,9 +39,9 @@ class CleanArchitectureSnippets {
             .assert { it.numDeclarations() == 1 && it.containsFunction("invoke") && it.isPublicOrDefault() }
     }
 
```

Comment:
I changed to `declarations()` because eg. in Mango we have interfaces meeting these conditions.

Reply:
Please revert this change, as this check is only for classes  - Spring `Repository` annotation can be only added to classes